lib/rubysl/rubysl-socket/lib/rubysl/socket/socket_options.rb

```diff
@@ -3,7 +3,7 @@ module Socket
     module SocketOptions
       def self.socket_level(level, family = nil)
         if level.is_a?(Symbol) or level.is_a?(String)
-          if ::Socket.const_defined?(level)
+          if ::Socket.const_defined?(level, false) # MODIFIED added inherit false
```

Comment:
I often use Truffle: or TruffleRuby: to mark changes, it would be better to be consistent (either way).